it/test/repositories/OptOutSessionDataRepositoryISpec.scala

```diff
@@ -51,7 +51,7 @@ class OptOutSessionDataRepositoryISpec extends ComponentSpecBase with ScalaFutur
           journeyType = OptOutJourney.Name,
           optOutSessionData = Some(expectedOptOutSessionData))
 
-        repository.set(expectedSessionData)
+        repository.set(expectedSessionData).futureValue
```

Comment:
I think we are using await for this kind of situation if we need blocking

Reply:
Done